tck/src/main/java/org/eclipse/microprofile/config/tck/ImplicitConverterTest.java

```diff
@@ -21,9 +21,13 @@
 
 import static org.eclipse.microprofile.config.tck.base.AbstractTest.addFile;
 
+import java.time.YearMonth;
```

Comment:
This test is also broken, ProcessAnnotatedType is not fired for fields in an Arquillian test.

Reply:
Which bit is broken?

Reply:
used the classes to hold the field injection instead. This must have something to do with the Arquillian integrator you are using.